language_server/src/lsp/feature_workspace_folders.cpp

```diff
@@ -120,4 +132,28 @@ void feature_workspace_folders::did_change_watched_files(const json&, const json
         paths.begin(), paths.end(), std::back_inserter(c_uris), [](const std::string& s) { return s.c_str(); });
     ws_mngr_.did_change_watched_files(c_uris.data(), c_uris.size());
 }
+
+void feature_workspace_folders::send_configuration_request()
+{
+    if (!response_)
+        return;
+    static const json config_request_args { { "items", { { { "section", "hlasm" } } } } };
+    static uint64_t config_request_number = 0;
```

Comment:
Any concerns regarding access from multiple threads?

Do the numbers need to be only unique or do they actually have to form an uninterrupted sequence?

Reply:
I made the variable a member variable, there was really no reason to make it static...

The numbers just need to be unique.